Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,22 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, the policy engine can be directed to select nodes via CIDR / ipBlock.
+This requires Cilium to be configured with ``--policy-cidr-match-mode=nodes`` or
+the equivalent Helm value ``policyCIDRMatchMode: nodes``. It is safe to toggle this
+option on a running cluster, nor does it affect upgrades or downgrades.
+
+When this is enabled, every node will have a distinct local :ref:`security identity <security_identities>`
+allocated for it, which will cause a minor increase in memory usage on very large clusters.
```

Comment:
Specific is better than general, use present tense
```suggestion
When CIDR/ipblock node selection is enabled, every node has a distinct local :ref:`security identity <security_identities>` allocated for it, which causes a minor increase in memory usage on very large clusters.
```

Reply:
reworded it to be more precise anyways, ptal